pkg/errors/errors.go

```diff
@@ -0,0 +1,9 @@
+package errors
```

Comment:
is this even needed?

Reply:
So while this is a single stem in the wind, for now, I was thinking a bit about how to work with errors more conceptually.

If you want to treat errors as consts, you need smth like the:
```go
type Error string
```
(as the Go's inbuilt `errors.errorString` can't be used that way as it's struct).

Now this is pretty convenient, and with improvements of Go's error handling since 1.13 allows you to easily use `errors.Is` to compare errors (incl. wrapped errors - that's what we do with the `fmt.Errorf("... %w", err)`.

I.e. we can easily check if returned error, or any of the wrapped errors inside, is of given type:
```go
if !errors.Is(err, kerrors.ERR_NOT_SUPPORTED) {
   ...
}
```

Finally, instead of doing this in individual packages, I was thinking at least the type should be central.